src/Commands/Site/LabelCommand.php

```diff
@@ -23,7 +23,9 @@ class LabelCommand extends SiteCommand
      *
      * @authorize
      *
-     * @command site:label
+     * @command site:label:set
+     * @alias site:label-set
```

Comment:
Maybe `set-label` would be a better compliment to `slbl`. The existing one is the same number of characters as the primary command; I think we don't need that.